internal/testbed/load/tests/metrics_test.go

```diff
@@ -247,6 +247,30 @@ func TestPrometheusMetric(t *testing.T) {
 			},
 			processors: defaultProcessors,
 		},
+		{
+			name:     "Prometheus Prometheus 1kDPS - 10 Prometheus Endpoints",
```

Comment:
prometheus prometheus? 😄

Reply:
prometh.... i mean, good catch - I adapted the test names now :)